datalake_common/metadata.py

```diff
@@ -172,8 +173,8 @@ def _normalize_dates(self):
 
     def _normalize_end(self):
         if 'end' not in self:
-            return
-        if self['end'] is not None:
+            self['end'] = None
+        elif self['end'] is not None:
```

Comment:
Can avoid extra lookups with `self.setdefault('end', None)` and modifying `self.normalize_date` to handle `None`. Also, can check file for use of `self.get('end')`, as those will hide cases where end is not set.

Reply:
good call